validation/mock_metric_validation.go

```diff
@@ -37,6 +37,13 @@ func IsValidRequest(req interface{}) error {
 		requestName = "GetMonitoredResourceDescriptorRequest"
 	case *monitoring.ListMetricDescriptorsRequest:
 		requestName = "ListMetricDescriptorsRequest"
+	case *monitoring.ListMonitoredResourceDescriptorsRequest:
+		requestName = "ListMonitoredResourceDescriptorsRequest"
+	case *monitoring.ListTimeSeriesRequest:
+		requestName = "ListTimeSeriesRequest"
```

Comment:
For `ListTimeSeriesRequest`, there are some additional restrictions (for example, `filter` and `interval` are required, while `order_by` must be left blank. Are we gonna add these in a future PR? (I'm fine with that since this PR is already pretty big)

Reply:
Definitely. I can add the required fields I missed in and leave the slightly more complicated validation for another PR.